nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -81,9 +81,27 @@
 
     protected String name;
 
+    /**
+     * @since 2.3
+     */
+    protected Boolean isProxy;
+
+    /**
+     * @deprecated since 2.3
+     */
     public Document(String id, String type, List<String> facets, String changeToken, String path, String state,
             String lockOwner, String lockCreated, String repositoryName, String versionLabel, String isCheckedOut,
             Map<String, Object> properties, Map<String, Object> contextParameters) {
+        this(id, type, facets, changeToken, path, state, lockOwner, lockCreated, repositoryName, versionLabel,
+                isCheckedOut, null, properties, contextParameters);
```

Comment:
Guess we can pass `false` if there is no `isProxy` argument as it won't be the case anymore on LTS 2016+